buddyboss/buddyboss-platform
# PROD-822 The Conversation doesn't fetch if the username registered with dot (.) inside the username

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2380  .

### How to test the changes in this Pull Request:

1. The Conversation doesn't fetch if the username registered with dot (.) inside the username
2. If you want to replicate the issue on your dev site then Create a user called "W.W" and see the next.
3. https://www.loom.com/share/018b3de5e4f14ac7885bdb5adfdb9198

### Proof Screenshots or Video

https://www.loom.com/share/3935db078ea3440cab7dac8be574b742

### Other information:
